src/gameplay/phases/common/votingMission.ts

```diff
@@ -153,6 +155,13 @@ class VotingMission implements IPhase {
             1 +
             this.thisRoom.playersInGame.length) %
           this.thisRoom.playersInGame.length;
+
+        if(!this.thisRoom.hasSinadRun && this.thisRoom.shouldSinadRun()) {
+          this.thisRoom.updateMissionSizesSinad();
```

Comment:
Can you rename this to `this.thisRoom.checkSinad()`, and then move the hasSinadRun check, and shouldSinadRun check into `checkSinad()`? 

That way it's a single entrypoint, making it easier for clients to use.

Reply:
Then you can remove `this.thisRoom.hasSinadRun = true;` below as well